github_discovery.py

```diff
@@ -614,6 +643,11 @@ def process_repo(**component):
   # Add versions to final data dict.
   data.update({'versions': versions_data})
 
+  # Add trivy scan result to final data dict.
+  data.update({'trivy_scan_summary': trivy_scan_summary})
```

Comment:
Should this line be futher up, under the `if cirlcleci_config:` block?

Reply:
Moved to the circleci block